### PR TITLE
Update code quality help note

### PR DESCRIPTION
**Pull request by jannaleong**
Improve code quality

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/HelpCommandParser.java [hunk 1]
```diff
@@ -13,19 +13,21 @@
...
+ * Parses input arguments and creates a new HelpCommand object.
  */
 public class HelpCommandParser implements Parser<HelpCommand> {
     private final Logger logger = LogsCenter.getLogger(getClass());
 
     /**
      * Parses the given {@code String} of arguments in the context of the HelpCommand
      * and returns a HelpCommand object for execution. Parameter {@code args} cannot be null.
-     * @throws ParseException If the user input does not conform to the expected format
+     *
```
Comment (yleeyilin) [inline src/main/java/seedu/address/logic/parser/HelpCommandParser.java:24]: Should not have a space here.

**Review by yleeyilin — APPROVED** [review]
Minor comments, that should be applied to all javadocs, see the javadocs example in the textbook. Otherwise, LGTM!

File: src/test/java/seedu/address/model/person/DeadlineNoteTest.java [hunk 2]
```diff
@@ -48,21 +46,24 @@ public void convertSuccessfulDate() {
 
     @Test
     public void equals() {
-        Note deadlineNote = new DeadlineNote("kind cats", "2025-10-10");
+        Note deadlineNote1 = new DeadlineNote("kind cats", "2025-10-10");
```
Comment (jamessinmaojun) [inline src/test/java/seedu/address/model/person/DeadlineNoteTest.java:49]: variable names should not contain numbers!

See [here](https://nus-cs2103-ay2324s2.github.io/website/schedule/week4/topics.html#w4-6-code-quality-naming)

**Review by jamessinmaojun — APPROVED** [review]
I'll fix the naming convention later, otherwise lgtm

**Pull request merged**